### ISSUE TITLE
Weird Symbols show up when using Local version compared to CDN

### ISSUE THREAD

**Issue by tkalp** [plotly/plotly.js#7052]
Hello,

I am currently trying to plot a graph using `plotly-cartesian.min.js`. We are currently limited to downloading and using a local copy rather than the CDN. I am plotting a graph that use the tickformat '.1e' for the y axis. 

When I run the code using the local version, I get some rather weird symbols when the values are between 0 and 1. It looks like there is some encoding issue but the html I am running has the tags `<meta charset="utf-8" />` as well as the script has the `charset` specified.

When I run the CDN version, the values are what are to be expected so I am just wondering if there is something else I need to specify in the configuration of either the web server we are running, or within the html itself.

You can reproduce this by using the following. 

1. Open this code pen <a href="https://codepen.io/tk-codes/pen/MWMKbjK">here</a>
2. You will see that the y-axis is formatted normally as expected.
3. Now instead of using the CDN, copy paste the `plotly-cartesian.min.js` contents between the <script> tags in the html.
4. You will then see the weird characters show up.

This is the weird symbols using the local version

<img width="396" alt="image" src="https://github.com/user-attachments/assets/e766c2ae-915e-4de2-b771-35cb082c0abe">

This is the normal version using the CDN

<img width="257" alt="image" src="https://github.com/user-attachments/assets/18476e36-4c5e-46ff-8c61-1f7f944be818">

**Comment by LonelyInstantN** [comment]
![image](https://github.com/user-attachments/assets/69ac401a-e386-4265-ba07-5d8922b9708e)
Exact same problem on me. Also using local version of `plotly-2.32.0.min.js`.

**Comment by LonelyInstantN** [comment]
> Hello,
> 
> I am currently trying to plot a graph using `plotly-cartesian.min.js`. We are currently limited to downloading and using a local copy rather than the CDN. I am plotting a graph that use the tickformat '.1e' for the y axis.
> 
> When I run the code using the local version, I get some rather weird symbols when the values are between 0 and 1. It looks like there is some encoding issue but the html I am running has the tags `<meta charset="utf-8" />` as well as the script has the `charset` specified.
> 
> When I run the CDN version, the values are what are to be expected so I am just wondering if there is something else I need to specify in the configuration of either the web server we are running, or within the html itself.
> 
> You can reproduce this by using the following.
> 
> 1. Open this code pen [here](https://codepen.io/tk-codes/pen/MWMKbjK)
> 2. You will see that the y-axis is formatted normally as expected.
> 3. Now instead of using the CDN, copy paste the `plotly-cartesian.min.js` contents between the <script> tags in the html.
> 4. You will then see the weird characters show up.
> 
> This is the weird symbols using the local version
> 
> <img alt="image" width="396" src="https://private-user-images.githubusercontent.com/93667579/349556159-e766c2ae-915e-4de2-b771-35cb082c0abe.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjIyMTc2NDgsIm5iZiI6MTcyMjIxNzM0OCwicGF0aCI6Ii85MzY2NzU3OS8zNDk1NTYxNTktZTc2NmMyYWUtOTE1ZS00ZGUyLWI3NzEtMzVjYjA4MmMwYWJlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MjklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzI5VDAxNDIyOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTI0MWFlODQyYWVmZjg3Njc2YjgwYTdiYzYyNGNlYzc5ZGE5MzE2M2I3MWQwZTVhZGI3OTJjZTM1NDlmNTBjMDcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.eNf6aan4ocZqYGLqMs1yy16m9EE2k8KPMMoAt9n3h98">
> This is the normal version using the CDN
> 
> <img alt="image" width="257" src="https://private-user-images.githubusercontent.com/93667579/349556279-18476e36-4c5e-46ff-8c61-1f7f944be818.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjIyMTc2NDgsIm5iZiI6MTcyMjIxNzM0OCwicGF0aCI6Ii85MzY2NzU3OS8zNDk1NTYyNzktMTg0NzZlMzYtNGM1ZS00NmZmLThjNjEtMWY3Zjk0NGJlODE4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MjklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzI5VDAxNDIyOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWZiMGE4MTU2OWNlOTQ3ZDQ2ZDUwOGIyMDU3YWRhMTNhOTZhMDY5ZjJlNjMwZDdhZTVhMWZmNWQ2YjgyOGQ4NGImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.QSFZ_q57v-V09tUl_WuRZyDS92zXLNsvhcS53iNy-XU">
Hey, I found out just add `<meta charset="utf-8">` to your html fix this problem;

**Comment by tkalp** [comment]
Thank you for the recommendation. Unfortunately, that does not fix our issue as mentioned in the original issue description. It may also be important to note that I  am using nginx in a self-hosted configuration with docker. I've tried to change the config settings in the container to use the UTF-8 encoding as well, but no luck.

**Comment by chris-ray** [comment]
hey @tkalp thanks for bringing this up. I found this because I had the same gibberish characters problem as you, but i didn't realize the CDN version worked. That gave me the idea the maybe if I downloaded a copy of the file using curl, rather than copying and pasting from the browser, it might fix it. And voila, it does.

tldr; don't copy and paste from the browser - use curl to download the file.
